Doesn't look good:
```
2020-07-27T16:13:56.3488679Z [ERROR] io.quarkus.it.infinispan.client.InfinispanClientFunctionalityTest.testNearCacheInvalidation  Time elapsed: 0.021 s  <<< ERROR!
2020-07-27T16:13:56.3493510Z java.lang.RuntimeException: java.lang.RuntimeException: Failed to start quarkus
2020-07-27T16:13:56.3499841Z 	at io.quarkus.test.junit.QuarkusTestExtension.throwBootFailureException(QuarkusTestExtension.java:470)
2020-07-27T16:13:56.3505653Z 	at io.quarkus.test.junit.QuarkusTestExtension.beforeEach(QuarkusTestExtension.java:352)
2020-07-27T16:13:56.3513751Z 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeBeforeEachCallbacks$1(TestMethodTestDescriptor.java:161)
2020-07-27T16:13:56.3522837Z 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeBeforeMethodsOrCallbacksUntilExceptionOccurs$5(TestMethodTestDescriptor.java:197)
2020-07-27T16:13:56.3529134Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-07-27T16:13:56.3539354Z 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeBeforeMethodsOrCallbacksUntilExceptionOccurs(TestMethodTestDescriptor.java:197)
2020-07-27T16:13:56.3560692Z 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeBeforeEachCallbacks(TestMethodTestDescriptor.java:160)
2020-07-27T16:13:56.3577031Z 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:131)
2020-07-27T16:13:56.3583329Z 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:71)
2020-07-27T16:13:56.3599605Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:135)
2020-07-27T16:13:56.3608037Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-07-27T16:13:56.3618055Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:125)
2020-07-27T16:13:56.3624170Z 	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:135)
2020-07-27T16:13:56.3631976Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:123)
2020-07-27T16:13:56.3638525Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-07-27T16:13:56.3644627Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:122)
2020-07-27T16:13:56.3652732Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:80)
2020-07-27T16:13:56.3658751Z 	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
2020-07-27T16:13:56.3668530Z 	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
2020-07-27T16:13:56.3675233Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:139)
2020-07-27T16:13:56.3681930Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-07-27T16:13:56.3688625Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:125)
2020-07-27T16:13:56.3693563Z 	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:135)
2020-07-27T16:13:56.3700036Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:123)
2020-07-27T16:13:56.3706332Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-07-27T16:13:56.3712810Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:122)
2020-07-27T16:13:56.3721668Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:80)
2020-07-27T16:13:56.3726840Z 	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
2020-07-27T16:13:56.3736129Z 	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
2020-07-27T16:13:56.3743659Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:139)
2020-07-27T16:13:56.3750038Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-07-27T16:13:56.3756815Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:125)
2020-07-27T16:13:56.3762294Z 	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:135)
2020-07-27T16:13:56.3768814Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:123)
2020-07-27T16:13:56.3774887Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-07-27T16:13:56.3782868Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:122)
2020-07-27T16:13:56.3788661Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:80)
2020-07-27T16:13:56.3797339Z 	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:32)
2020-07-27T16:13:56.3804135Z 	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
2020-07-27T16:13:56.3810582Z 	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:51)
2020-07-27T16:13:56.3815731Z 	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:248)
2020-07-27T16:13:56.3821575Z 	at org.junit.platform.launcher.core.DefaultLauncher.lambda$execute$5(DefaultLauncher.java:211)
2020-07-27T16:13:56.3827474Z 	at org.junit.platform.launcher.core.DefaultLauncher.withInterceptedStreams(DefaultLauncher.java:226)
2020-07-27T16:13:56.3830419Z 	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:199)
2020-07-27T16:13:56.3832268Z 	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:132)
2020-07-27T16:13:56.3838652Z 	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.execute(JUnitPlatformProvider.java:188)
2020-07-27T16:13:56.3845054Z 	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:154)
2020-07-27T16:13:56.3852382Z 	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:128)
2020-07-27T16:13:56.3857825Z 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
2020-07-27T16:13:56.3863043Z 	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
2020-07-27T16:13:56.3867738Z 	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
2020-07-27T16:13:56.3874817Z 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
2020-07-27T16:13:56.3881715Z Caused by: java.lang.RuntimeException: Failed to start quarkus
2020-07-27T16:13:56.3886361Z 	at io.quarkus.runner.ApplicationImpl.doStart(ApplicationImpl.zig:615)
2020-07-27T16:13:56.3890454Z 	at io.quarkus.runtime.Application.start(Application.java:90)
2020-07-27T16:13:56.3912045Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2020-07-27T16:13:56.3913128Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2020-07-27T16:13:56.3914095Z 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2020-07-27T16:13:56.3914989Z 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
2020-07-27T16:13:56.3916011Z 	at io.quarkus.runner.bootstrap.StartupActionImpl.run(StartupActionImpl.java:223)
2020-07-27T16:13:56.3916926Z 	at io.quarkus.test.junit.QuarkusTestExtension.doJavaStart(QuarkusTestExtension.java:209)
2020-07-27T16:13:56.3917823Z 	at io.quarkus.test.junit.QuarkusTestExtension.ensureStarted(QuarkusTestExtension.java:448)
2020-07-27T16:13:56.3919908Z 	at io.quarkus.test.junit.QuarkusTestExtension.beforeAll(QuarkusTestExtension.java:481)
2020-07-27T16:13:56.3920847Z 	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$invokeBeforeAllCallbacks$7(ClassBasedTestDescriptor.java:359)
2020-07-27T16:13:56.3922034Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-07-27T16:13:56.3922973Z 	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeBeforeAllCallbacks(ClassBasedTestDescriptor.java:359)
2020-07-27T16:13:56.3923877Z 	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:189)
2020-07-27T16:13:56.3924770Z 	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:78)
2020-07-27T16:13:56.3925664Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:132)
2020-07-27T16:13:56.3926540Z 	... 32 more
2020-07-27T16:13:56.3927457Z Caused by: org.infinispan.client.hotrod.exceptions.HotRodClientException:Request for messageId=15 returned server error (status=0x85): org.infinispan.commons.marshall.MarshallingException: No marshaller registered for Protobuf type book_sample.Book
2020-07-27T16:13:56.3928388Z 	at org.infinispan.client.hotrod.impl.protocol.Codec20.checkForErrorsInResponseStatus(Codec20.java:329)
2020-07-27T16:13:56.3932519Z 	at org.infinispan.client.hotrod.impl.protocol.Codec20.readHeader(Codec20.java:168)
2020-07-27T16:13:56.3939391Z 	at org.infinispan.client.hotrod.impl.transport.netty.HeaderDecoder.decode(HeaderDecoder.java:140)
2020-07-27T16:13:56.3946779Z 	at org.infinispan.client.hotrod.impl.transport.netty.HintedReplayingDecoder.callDecode(HintedReplayingDecoder.java:94)
2020-07-27T16:13:56.3952876Z 	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:276)
2020-07-27T16:13:56.3959671Z 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
2020-07-27T16:13:56.3966087Z 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
2020-07-27T16:13:56.3972355Z 	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
2020-07-27T16:13:56.3977528Z 	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:286)
2020-07-27T16:13:56.3984414Z 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
2020-07-27T16:13:56.3991285Z 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
2020-07-27T16:13:56.3999175Z 	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
2020-07-27T16:13:56.4004236Z 	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1518)
2020-07-27T16:13:56.4009104Z 	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1267)
2020-07-27T16:13:56.4013175Z 	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1314)
2020-07-27T16:13:56.4019854Z 	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:501)
2020-07-27T16:13:56.4025162Z 	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:440)
2020-07-27T16:13:56.4031464Z 	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:276)
2020-07-27T16:13:56.4040724Z 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
2020-07-27T16:13:56.4047139Z 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
2020-07-27T16:13:56.4053383Z 	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
2020-07-27T16:13:56.4059389Z 	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)
2020-07-27T16:13:56.4065730Z 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
2020-07-27T16:13:56.4072103Z 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
2020-07-27T16:13:56.4078912Z 	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)
2020-07-27T16:13:56.4085965Z 	at io.netty.channel.epoll.AbstractEpollStreamChannel$EpollStreamUnsafe.epollInReady(AbstractEpollStreamChannel.java:792)
2020-07-27T16:13:56.4092398Z 	at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:475)
2020-07-27T16:13:56.4098343Z 	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:378)
2020-07-27T16:13:56.4104250Z 	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)
2020-07-27T16:13:56.4109052Z 	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
2020-07-27T16:13:56.4114656Z 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
2020-07-27T16:13:56.4121130Z 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
2020-07-27T16:13:56.4124714Z 	at java.base/java.lang.Thread.run(Thread.java:834)
```

Did you run the tests locally before pushing a PR? You can quickly deploy things with `mvn clean install -Dquickly` and the run your specific IT.

--- a/bom/application/pom.xml
+++ b/bom/application/pom.xml
@@ -123,8 +123,8 @@
         <testng.version>6.14.2</testng.version>
         <assertj.version>3.16.1</assertj.version>
         <json-smart.version>2.3</json-smart.version>
-        <infinispan.version>10.1.5.Final</infinispan.version>
-        <infinispan.protostream.version>4.3.2.Final</infinispan.protostream.version>
+        <infinispan.version>11.0.1.Final</infinispan.version>
+        <infinispan.protostream.version>4.3.3.Final</infinispan.protostream.version>
         <caffeine.version>2.8.5</caffeine.version>
         <netty.version>4.1.49.Final</netty.version>
         <reactive-streams.version>1.0.3</reactive-streams.version>

--- a/extensions/infinispan-client/deployment/src/main/java/io/quarkus/infinispan/client/deployment/InfinispanClientProcessor.java
+++ b/extensions/infinispan-client/deployment/src/main/java/io/quarkus/infinispan/client/deployment/InfinispanClientProcessor.java
@@ -20,7 +20,7 @@ import org.infinispan.client.hotrod.exceptions.HotRodClientException;
 import org.infinispan.client.hotrod.impl.ConfigurationProperties;
 import org.infinispan.client.hotrod.logging.Log;
 import org.infinispan.client.hotrod.logging.LogFactory;
-import org.infinispan.client.hotrod.marshall.ProtoStreamMarshaller;
+import org.infinispan.commons.marshall.ProtoStreamMarshaller;
 import org.infinispan.commons.util.Util;
 import org.infinispan.protostream.BaseMarshaller;
 import org.infinispan.protostream.EnumMarshaller;

--- a/extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/InfinispanClientProducer.java
+++ b/extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/InfinispanClientProducer.java
@@ -25,8 +25,8 @@ import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
 import org.infinispan.client.hotrod.impl.ConfigurationProperties;
 import org.infinispan.client.hotrod.logging.Log;
 import org.infinispan.client.hotrod.logging.LogFactory;
-import org.infinispan.client.hotrod.marshall.ProtoStreamMarshaller;
 import org.infinispan.commons.marshall.Marshaller;
+import org.infinispan.commons.marshall.ProtoStreamMarshaller;
 import org.infinispan.commons.util.Util;
 import org.infinispan.counter.api.CounterManager;
 import org.infinispan.protostream.BaseMarshaller;
@@ -296,10 +296,10 @@ public class InfinispanClientProducer {
     }
 
     /**
-     * Retrieves the deprecated {@link Remote} annotation instance from the set
+     * Retrieves the deprecated {@link io.quarkus.infinispan.client.Remote} annotation instance from the set
      *
      * @param annotationSet the annotation set.
-     * @return the {@link Remote} annotation instance or {@code null} if not found.
+     * @return the {@link io.quarkus.infinispan.client.Remote} annotation instance or {@code null} if not found.
      */
     private io.quarkus.infinispan.client.Remote getRemoteAnnotation(Set<Annotation> annotationSet) {
         for (Annotation annotation : annotationSet) {

--- a/extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/graal/SubstituteConfigurationBuilder.java
+++ b/extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/graal/SubstituteConfigurationBuilder.java
@@ -4,7 +4,6 @@ import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
 import org.infinispan.client.hotrod.marshall.BytesOnlyMarshaller;
 import org.infinispan.commons.marshall.Marshaller;
 
-import com.oracle.svm.core.annotate.Alias;
 import com.oracle.svm.core.annotate.Substitute;
 import com.oracle.svm.core.annotate.TargetClass;
 
@@ -15,11 +14,9 @@ import com.oracle.svm.core.annotate.TargetClass;
  */
 @TargetClass(ConfigurationBuilder.class)
 public final class SubstituteConfigurationBuilder {
-    @Alias
-    private Marshaller marshaller;
 
     @Substitute
-    private void handleNullMarshaller() {
-        marshaller = BytesOnlyMarshaller.INSTANCE;
+    private Marshaller handleNullMarshaller() {
+        return BytesOnlyMarshaller.INSTANCE;
     }
 }

--- a/extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/graal/SubstituteRemoteCacheImpl.java
+++ b/extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/graal/SubstituteRemoteCacheImpl.java
@@ -2,6 +2,7 @@ package io.quarkus.infinispan.client.runtime.graal;
 
 import javax.management.ObjectName;
 
+import org.infinispan.client.hotrod.configuration.Configuration;
 import org.infinispan.client.hotrod.impl.RemoteCacheImpl;
 import org.infinispan.client.hotrod.impl.operations.OperationsFactory;
 import org.infinispan.commons.marshall.Marshaller;
@@ -31,7 +32,7 @@ public final class SubstituteRemoteCacheImpl {
     // Sadly this method is public, so technically a user could get a Runtime error if they were referencing
     // it before - but it is the only way to make graal happy
     @Delete
-    public void init(Marshaller marshaller, OperationsFactory operationsFactory, int estimateKeySize,
-            int estimateValueSize, int batchSize, ObjectName jmxParent) {
+    public void init(Marshaller marshaller, OperationsFactory operationsFactory,
+            Configuration configuration, ObjectName jmxParent) {
     }
 }

--- a/extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/graal/SubstituteRemoteCacheManager.java
+++ b/extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/graal/SubstituteRemoteCacheManager.java
@@ -2,7 +2,7 @@ package io.quarkus.infinispan.client.runtime.graal;
 
 import org.infinispan.client.hotrod.RemoteCacheManager;
 import org.infinispan.client.hotrod.configuration.Configuration;
-import org.infinispan.client.hotrod.impl.RemoteCacheImpl;
+import org.infinispan.client.hotrod.impl.InternalRemoteCache;
 import org.infinispan.client.hotrod.impl.operations.OperationsFactory;
 import org.infinispan.commons.marshall.Marshaller;
 
@@ -23,10 +23,9 @@ public final class SubstituteRemoteCacheManager {
     private Configuration configuration;
 
     @Substitute
-    private void initRemoteCache(RemoteCacheImpl remoteCache, OperationsFactory operationsFactory) {
+    private void initRemoteCache(InternalRemoteCache<?, ?> remoteCache, OperationsFactory operationsFactory) {
         // Invoke the init method that doesn't have the JMX ObjectName argument
-        remoteCache.init(marshaller, operationsFactory, configuration.keySizeEstimate(),
-                configuration.valueSizeEstimate(), configuration.batchSize());
+        remoteCache.init(marshaller, operationsFactory, configuration);
     }
 
     @Substitute

--- a/integration-tests/infinispan-client/pom.xml
+++ b/integration-tests/infinispan-client/pom.xml
@@ -95,17 +95,17 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-core</artifactId>
-            <version>2.10.0</version>
+            <version>2.11.0</version>
         </dependency>
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>2.10.0</version>
+            <version>2.11.0</version>
         </dependency>
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-annotations</artifactId>
-            <version>2.10.0</version>
+            <version>2.11.0</version>
         </dependency>
 
     </dependencies>

--- a/integration-tests/infinispan-client/src/test/java/io/quarkus/it/infinispan/client/InfinispanServerTestResource.java
+++ b/integration-tests/infinispan-client/src/test/java/io/quarkus/it/infinispan/client/InfinispanServerTestResource.java
@@ -3,6 +3,7 @@ package io.quarkus.it.infinispan.client;
 import java.util.Collections;
 import java.util.Map;
 
+import org.infinispan.commons.dataconversion.MediaType;
 import org.infinispan.commons.test.TestResourceTracker;
 import org.infinispan.configuration.cache.ConfigurationBuilder;
 import org.infinispan.configuration.global.GlobalConfigurationBuilder;
@@ -23,10 +24,14 @@ public class InfinispanServerTestResource implements QuarkusTestResourceLifecycl
     @Override
     public Map<String, String> start() {
         TestResourceTracker.setThreadTestName("InfinispanServer");
+        ConfigurationBuilder configurationBuilder = new ConfigurationBuilder();
+        configurationBuilder.encoding().mediaType(MediaType.APPLICATION_PROTOSTREAM_TYPE);
         EmbeddedCacheManager ecm = TestCacheManagerFactory.createCacheManager(
                 new GlobalConfigurationBuilder().nonClusteredDefault().defaultCacheName("default"),
-                new ConfigurationBuilder());
-        ecm.defineConfiguration("magazine", new ConfigurationBuilder().build());
+                configurationBuilder);
+
+        ecm.defineConfiguration("magazine", configurationBuilder.build());
+
         // Client connects to a non default port
         final HotRodServerConfigurationBuilder hotRodServerConfigurationBuilder = new HotRodServerConfigurationBuilder();
         hotRodServerConfigurationBuilder

